modules/ravestate_telegramio/telegram_bot.py

```diff
@@ -0,0 +1,73 @@
+from typing import Set
+
+from ravestate.context import Context
+from ravestate.receptor import receptor
+from ravestate.state import state
+from ravestate.wrappers import ContextWrapper
+from telegram import Bot, Update, TelegramError
+from telegram.ext import Updater, MessageHandler, Filters, Dispatcher
+import logging
+
+# TODO Put the token of your bot here
+TELEGRAM_BOT_TOKEN: str = None
```

Comment:
Correct me if I'm wrong, but don't we want the Token to be loaded from an `API.Key` file that's gitignored? That way we could make sure that we don't accidentally upload the keys

Reply:
Do we already have a location/structure for this API.key file? @josephbirkner

Reply:
I would say that we should keep it similar to how `roboy_dialog` did and use a YAML formated document called API.key where all keys are stored. YAML has a bunch of parsers written for it like https://pyyaml.org/wiki/PyYAML

Reply:
implemented getting the key from the API.key file